server/conn.go

```diff
@@ -37,50 +37,43 @@ type Conn struct {
 }
 
 var baseConnID uint32 = 10000
+var defaultServer *Server
 
 // NewConn: create connection with default server settings
```

Comment:
🎨 nitpicking

 You would break the godoc comment if you do like that

This is the expected format

```suggestion
 // NewConn: create connection with default server settings
//
// Deprecated: Use Server.NewConn instead.
```

You can see the difference by using [pkgsite](https://github.com/golang/pkgsite) locally